### PR TITLE
dashbaords data and compare dashbaord

### PR DESCRIPTION
**Pull request by Sindhu15**

### CODE REVIEW

File: components/Header/index.js [hunk 1]
```diff
@@ -5,32 +5,34 @@ const metricsData = require("../../data/metrics.json");
...
   return {
-    id: 1,
-    selectedRange: 1
-  }
+    id, 
+    range
+  };
 }
 
 export const getMetricData = () => {
```
Comment (Saajan) [inline components/Header/index.js:30]: This is not required actually

File: components/Header/index.js [hunk 2]
```diff
@@ -5,32 +5,34 @@ const metricsData = require("../../data/metrics.json");
...
+    case 'yesterday':
+      return 2;
+    case 'week':
+      return 3;
+    case 'month':
+      return 4;
+    default:
+      return 1;
   }
-  const obj = metricsData.find(o => o.date === String(date));
```
Comment (Saajan) [inline components/Header/index.js:31]: this will get the metrics data for selected range

Comment (Sindhu15) [reply components/Header/index.js:31]: dont we have to consider the id?

Comment (Saajan) [reply components/Header/index.js:31]: What id?

**Pull request merged**